src/injected/components/details-dialog.tsx

```diff
@@ -38,7 +34,7 @@ export type DetailsDialogDeps = {
     targetPageActionMessageCreator: TargetPageActionMessageCreator;
     browserAdapter: BrowserAdapter;
 } & CopyIssueDetailsButtonDeps &
```

Comment:
>CopyIssueDetailsButtonDeps [](start = 4, length = 26)

`CommandBarDeps` includes this already, no?

Reply:
It does... bad merge on my side.